index.js

```diff
@@ -31,21 +32,35 @@ async function main(options) {
     modelName: options.model ? options.model : 'gpt-4-1106-preview',
   });
 
+  if (options.outputFilePath) {
+    createTestFile(options.outputFilePath);
+  }
+
   // eslint-disable-next-line no-constant-condition
   while (true) {
     const {task} = await prompt.get({
       properties: {
         task: {
           message: ' Input a task\n',
-          required: true,
+          required: false,
         },
       },
     });
+
+    if (task === '') {
+      if (options.outputFilePath) {
+        completeTestFile(options.outputFilePath);
+      }
```

Comment:
This is probably better done on the exit event so we also complete the file even if we CTRL+C the program.

Eg:
```
process.on('exit', () => {
     if (options.outputFilePath) {
        completeTestFile(options.outputFilePath);
      }
})
```

See https://nodejs.org/api/process.html#event-exit

Reply:
I tried this initially but i was for event `SIGINT` and it didn't work, ill try `exit` instead

Reply:
Awesome this worked, I should have tried `exit` instead of `SIGINT`
https://github.com/mayt/BrowserGPT/pull/16/commits/c7d903968752abd1a1492e542bfcc1ff6fd77953